### ISSUE TITLE
Is there something wrong with the use counting of deprecated (or all?) features?

### ISSUE THREAD

**Issue by mfreed7** [GoogleChrome/chromium-dashboard#1545]
Here are a couple example use counter plots:

- The `<content>` element:
https://chromestatus.com/metrics/feature/timeline/popularity/1896

- The HTML `<shadow>` element:
https://chromestatus.com/metrics/feature/timeline/popularity/1897

- The Custom Elements v0 API:
https://chromestatus.com/metrics/feature/timeline/popularity/457

- The HTML Imports feature:
https://chromestatus.com/metrics/feature/timeline/popularity/455

All of these were [removed from the Chromium codebase in December 2020](https://chromium-review.googlesource.com/c/chromium/src/+/2586019), yet they have very high (27% for `<content>`!!) usage, and bounce around **very** strangely. For example, `<content>` bounces from 1.6% in February to 27% in August. The `<shadow>` element went to near-zero in June, and is back to 0.3% in August. 

I am wondering if, perhaps, there is a "denominator" problem when counting these APIs? I.e. is it possible that for some reason the total page load count is stale or something? I would imagine that there is a relatively small group of people/bots using older versions of Chromium, and those might be encountering these elements and reporting usage. However, looking at, e.g. `<content>`, that would indicate that in August, 2021, over 25% (minimum!) of client reports were using a Chrome version < M89.

**Comment by mfreed7** [comment]
+@vmpstr FYI

> The above is only a theory, though, and until we know what the actual root cause is, I think all use counter metrics are likely suspect.

@jstenback Gentle ping - I am increasingly suspicious of the numbers in the use counter plots, which we use for decision making. I'm hoping this particular issue only applies to deprecated features, which would reduce the impact. However, this came up in a recent conversation about the `<details>` element, for which the [use counter data shows a precipitous drop recently](https://chromestatus.com/metrics/feature/timeline/popularity/480). Should we trust that drop?

**Comment by mfreed7** [comment]
Coming back to this issue, here's another one:

https://chromestatus.com/metrics/feature/timeline/popularity/2502

That hasn't been in Chromium code since about M79, yet it still shows 0.4% use counter, which is rising quite quickly.


Side-note: this issue would be fixed by fixing one of these two:
- https://github.com/GoogleChrome/chromium-dashboard/issues/2563
- https://github.com/GoogleChrome/chromium-dashboard/issues/2565

I'm a bit suspicious of the last one of those, since it offers a checkbox to show "obsolete" use counters. This bug essentially describes those as having fairly nebulous accuracy, so I'm not sure whether you want to show them.

@jrobbins are you the right person to tag here?

**Comment by jrobbins** [comment]
Yes, tagging me is fine.  I had written up #2565, but never got around to working on it.  Thanks for the reminder.

I don't understand what is happening in that graph for `<details>`.   That seems like a different problem.
